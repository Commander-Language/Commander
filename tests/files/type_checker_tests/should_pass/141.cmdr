//Test that not equals operation type checks
x = 1.1 != 2;
