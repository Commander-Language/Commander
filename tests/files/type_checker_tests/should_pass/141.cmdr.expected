(PrgmNode
	(ExprStmtNode (BinOpExprNode (BindingNode x) SET (BinOpExprNode (FloatExprNode 1.100000 FLOAT) NOT_EQUAL (IntExprNode 2 INT) BOOL) BOOL)))
